### PR TITLE
Implement RPC methods 

### PR DESCRIPTION
**Pull request by s1ntaxe770r**

### CODE REVIEW

File: application/container-builder/internal/containers/buildpack/builder.go [hunk 1]
```diff
@@ -3,43 +3,48 @@ package image
...
 type BuildOpt struct {
+	BuildID   string
 	RepoURL   string
 	ImageName string
 	BuildPack string
 	Builder   string
 }
 
+// BuildRequest is the request to build a container
 type BuildRequest struct {
```
Comment (Klaven) [inline application/container-builder/internal/containers/buildpack/builder.go:26]: IMO. This is not a request. you don't generally give a "status" in the request. the status is something that could be returned but that is it.

If this was me I would pull all DB models out into their own "models" section and don't reuse these models for other things as this gives us the most flexibility with how we store data on disk. it's also then easier to find where your database schema is :)

**Review by Klaven — COMMENTED** [review]
good start. 

A few little comments here. nothing major.

At a high level I think this could be 2 PR's, one to add the integration for sqlite and postgress with bun and another one to update the RPC calls. And maybe a third one to update the logging to apex.

File: proto/container/container.proto [hunk 2]
```diff
@@ -21,28 +21,41 @@ package container;
...
-	rpc ImageStatus (ContainerID) returns (ContainerStatus){  }
+  rpc ImageStatus (Build) returns (ContainerStatusReply){  }
 }
 
 
-
+// Enum of ContainerStatus
+enum ContainerStatus {
+  NOT_STARTED = 0;
+  WAITING = 1;
```
[the remark this answers is not in the document]
Comment (s1ntaxe770r) [reply proto/container/container.proto:31]: Waiting was supposed to be for future purposes when the implementation is a job queue? Or I also think waiting could be used for retries of failed builds

File: application/container-builder/internal/containers/buildpack/builder.go (hunk 1 above)
Comment (s1ntaxe770r) [reply application/container-builder/internal/containers/buildpack/builder.go:26]: Agreed. I wasn’t sure if it needed to be its own package but this definitely makes sense

**Review by Klaven — APPROVED** [review]
/lgtm

**Pull request merged**